packages/camera/src/testUtils/wrapper-utils.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { I18nextProvider, initReactI18next } from 'react-i18next';
+import { render } from '@testing-library/react-native';
+
+const wrapInRouter = (component) => (
+  <I18nextProvider i18={initReactI18next()}>
+    {component}
+  </I18nextProvider>
+);
+
+const initiateComponent = (ui, renderOptions) => {
+  if (!renderOptions) { return ui; }
+  let component = ui;
+  if (renderOptions.withI18) { component = wrapInRouter(component, renderOptions); }
```

Comment:
Instead of `wrapInRouter` name, plz use a specific name for function as we may create multiple functions for different types of wrappers.